featuremanagement/_models/_variant.py

```diff
@@ -13,22 +14,22 @@ class Variant:
     :param dict configuration: The configuration of the variant.
     """
 
-    def __init__(self, name, configuration):
+    def __init__(self, name: str, configuration: Union[Any, Dict]) -> None:
         self._name = name
         self._configuration = configuration
 
     @property
-    def name(self):
+    def name(self) -> str:
         """
         The name of the variant.
         :rtype: str
         """
         return self._name
 
     @property
-    def configuration(self):
+    def configuration(self) -> Union[Any, Dict]:
```

Comment:
Does a Union here provide any benefit- since one of the types is already Any?

Reply:
This is the variant object, so I was sort of thinking it made sense sort of as it could be a config or a json. But i'm good making it just Any.